js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -227,17 +235,20 @@ const loadVillageMain = () => {
             Zone.getZones().village.field
         ));
 
+        const easternBorderLocked = STORY.villageAttackEnded ? false : true;
         GAME.addEntity(new Border(
             new Vector(ZONE.MAX_PT.x, 0),
             new Vector(1, ZONE.PIXEL_SIZE.y),
-            Zone.getZones().village.hillDownFromMain
+            Zone.getZones().village.hillDownFromMain,
+            easternBorderLocked
         ));
+
         let weather = "warm";
-        let surfaceSnow = false;
-        if (weather === "snow") {
-            surfaceSnow = true
-        }
-        TilemapInterpreter.setTilemap(villageMainTileMap, surfaceSnow);
+        // let surfaceSnow = false;
+        // if (weather === "snow") {
+        //     surfaceSnow = true
+        // }
+        TilemapInterpreter.setTilemap(villageMainTileMap, false);
```

Comment:
Doesn't this default to false? If not then we could update that so you don't have to write that extra variable.